app/helpers/application_helper.rb

```diff
@@ -3,6 +3,10 @@
 require "cgi"
 
 module ApplicationHelper
+  def humanized_date(datetime)
```

Comment:
I came across this method the other day, it's [a nice one from active support](https://apidock.com/rails/ActiveSupport/CoreExtensions/Date/Conversions/to_formatted_s):
```
Date.new(2007, 11, 10).to_formatted_s(:long)
=> "November 10, 2007"
```
The cool bit is you can define custom formats defined as symbols in an initaliszer too:

```
  # config/initializers/time_formats.rb
  Date::DATE_FORMATS[:month_and_year] = "%B %Y"
  Date::DATE_FORMATS[:short_ordinal] = lambda { |date| date.strftime("%B #{date.day.ordinalize}") }
```

Non blocking but might be neat if we could move this to an inine active support call?

Reply:
That's pretty nice, it ensures we get our date formats consistent everywhere

Reply:
Hmm, I've tried it and it seems to be falling back to a default format.  I'll leave it for now.